docs/types.rst

```diff
@@ -353,6 +353,9 @@ bundles, listed in the order that they should be invoked:
    cryptographic signatures to authorize spending the inputs. You do not
    need to invoke this method if the bundle does not contain any
    transactions that spend IOTAs.
+-  ``add_signature_or_message: (List[Fragment], int) -> None``:
+   Adds signature or message fragments to transactions in the bundle
+   starting from ``start_index``.
```

Comment:
❔ Once a bundle is finalised, it's not really meant to accept further changes.  Also, calling this method after `sign_inputs` could potentially cause signatures to get overwritten.  Maybe consider moving this method to occur before `finalize()` in the sequence?

Reply:
got you, makes sense.